superset-frontend/src/explore/components/ChartPills.tsx

```diff
@@ -22,11 +22,11 @@ import RowCountLabel from 'src/explore/components/RowCountLabel';
 import CachedLabel from 'src/components/CachedLabel';
 import Timer from 'src/components/Timer';
 
-enum CHART_STATUS_MAP {
-  failed = 'danger',
-  loading = 'warning',
-  success = 'success',
-}
+const CHART_STATUS_MAP = {
+  failed: 'danger',
+  loading: 'warning',
+  success: 'success',
+};
```

Comment:
This was really not a proper enum or used as such, so I changed it to a regular object.